### PR TITLE
add unit test for lambda

### PR DESCRIPTION
**Pull request by ziyangczi**
added unit test for lambda

the rest of the code in lambda is heavy IO, so adding mocks as unit tests for them doesn't make much sense

### CODE REVIEW

File: backend/_tests/test_lambda.py [hunk 1]
```diff
@@ -0,0 +1,129 @@
+from unittest import mock
+import importlib
+import requests
+from requests.exceptions import HTTPError
+
+api = importlib.import_module("backend.lambda")
```
Comment (codemonkey800) [inline backend/_tests/test_lambda.py:6]: Is there a reason why you're using `importlib.import_module()` instead of `import api from backend.lambda`?

**Review by codemonkey800 — APPROVED** [review]
LGTM! I did have one question about `importlib`

File: backend/_tests/test_lambda.py [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+class FakeResponse:
+    def __init__(self, *, data: str):
+        self.text = data
+        self.status_code = requests.codes.ok
+
+    @property
+    def status_code(self):
+        status_code = self._status_code
+        self.status_code = requests.codes.ok + 100
```
Comment (justinelarsen) [inline backend/_tests/test_lambda.py:17]: requests.codes.ok + 100? Interesting way of doing this

**Review by justinelarsen — APPROVED** [review]
Looks good to me too!

File: backend/_tests/test_lambda.py (hunk 1 above)
Comment (ziyangczi) [reply backend/_tests/test_lambda.py:6]: nice catch! lambda is a keyword in python, so you can't really import from a file or module named lambda, so I went around with the string import method

**Pull request merged**